ROADMAP.md

```diff
@@ -0,0 +1,154 @@
+## Vision
+
+> Amethyst aims to be a modular, parallel and data-driven game engine
+
+**Modularity** The game engine should be easily extendable to fit the needs of the game developers for their game idea. Therefore we expose an entity-component system, which allows to define new components and processors without needing to change the game engine code. This should emphasize the 'game engine as library' aspect.
```

Comment:
> (...) do we want to pick a particular option as default while still leaving the option of switching it out for something else (...)

Do you mean that we pick what language to use or an actual script doing somehing on default?